node/service/src/lib.rs

```diff
@@ -277,13 +279,23 @@ where
 	))
 }
 
+// If we're using prometheus, use a registry with a prefix of `moonbeam`.
+fn set_prometheus_registry(config: &mut Configuration) -> Result<(), ServiceError> {
+	if let Some(PrometheusConfig { registry, .. }) = config.prometheus_config.as_mut() {
+		let labels = iter::once((String::from("chain"), config.chain_spec.id().into())).collect();
```

Comment:
It would be more readable to explicitly declare a HashMap rather than using an iterator.
You can use the crate [maplit](https://docs.rs/maplit/1.0.2/maplit/) for that.

Reply:
maplit is already in the project dependencies, so the impact on the compilation time is more than negligible, we privilege the code readability :)